schema/transportation/segment.yaml

```diff
@@ -26,6 +26,15 @@ properties:
           properties:
             class: { "$ref": "#/$defs/propertyDefinitions/roadClass" }
             road: { "$ref": "#/$defs/propertyDefinitions/road" }
```

Comment:
i would include a comment in `road` property definition's description that it is a deprecated property in favor of top-level properties corresponding to its sub-properties